### PR TITLE
Feature/container form

### PR DESCRIPTION
**Pull request by TeddyAndersson**
First implementation of The Form component.

The Form Container Component allows you to create, process and reuse forms. The Form component is a tool to help you solve the problem of allowing end-users to interact with the data and modify the data in your application.

### CODE REVIEW

File: source/components/atoms/Stepper/Stepper.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { View } from 'react-native';
```
Comment (mysticlagoona) [inline source/components/atoms/Stepper/Stepper.js:3]: Remove View as it is not used

Comment (mysticlagoona) [reply source/components/atoms/Stepper/Stepper.js:3]: Simulator Screen Shot - iPhone 11 - 2020-06-02 at 11 42 32

**Comment by mysticlagoona** [comment]
Simulator Screen Shot - iPhone 11 - 2020-06-02 at 11 42 54

**Comment by mysticlagoona** [comment]
I am able to drag the form up and down after the footer .

**Comment by TeddyAndersson** [comment]
Yeah i think thats a behavior that we want? :)

**Comment by TeddyAndersson** [comment]
If you mean the bounce effect?

**Comment by mysticlagoona** [comment]
Do we !! :) 
yes bounce effect .

**Comment by mysticlagoona** [comment]
May be make a todo to take the issue later

**Comment by TeddyAndersson** [comment]
I think that Gustav wanted to support the Bounce effect. Can you create a post it?

**Pull request merged**